src/filesystem.rs

```diff
@@ -26,12 +26,30 @@ fn paths_from_path_param(env_var: &str) -> impl Iterator<Item = &str> {
     env_var.split(':').filter(|folder| folder != &"")
 }
 
+fn compiled_default_path(path: Option<&str>) -> Option<PathBuf> {
+    match path {
+        Some(path) => {
+            let path = Path::new(path);
+            if path.exists() {
+                Some(path.to_path_buf())
+            } else {
+                None
+            }
+        }
+        None => None,
+    }
+}
+
 pub fn default_cheat_pathbuf() -> Result<PathBuf> {
     let base_dirs = BaseDirs::new().ok_or_else(|| anyhow!("Unable to get base dirs"))?;
-
     let mut pathbuf = PathBuf::from(base_dirs.data_dir());
     pathbuf.push("navi");
     pathbuf.push("cheats");
+    if !pathbuf.exists() {
+        if let Some(path) = compiled_default_path(option_env!("NAVI_CHEATS_DIRECTORY")) {
```

Comment:
Instead of creating a new env var, could we please reuse https://github.com/denisidoro/navi/blob/8725654a746fa8c999fbd5437559fdfcf1f7fb53/src/env_var.rs#L14?

We would need to split it by `:` and get the first element.

Reply:
Hi @denisidoro. Certainly, happy to replace both env var names with the existing ones.

Just to be sure: both existing env vars are currently used at *runtime* to override / set the respective paths for navi to look into. This CL would use them at  *compile time* for the slightly different purpose of setting a fallback for the respective paths, in case they do not exist (i.e. if there is a system wide installation then navi will use that - if the users has not decided to set up a custom one in their home directory). So the env vars would be a bit "dual purpose".